code/graphics/grstub.cpp

```diff
@@ -48,11 +48,6 @@ int gr_stub_save_screen()
 	return 1;
 }
 
-int gr_stub_tcache_set(int bitmap_id, int bitmap_type, float *u_scale, float *v_scale, int tex_unit = 0)
```

Comment:
Ah, so the problem wasn't with the thing it was doing, but in exposing it to external code. Gotcha.